Personal Portfolio

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Uohna Thiessen</title>
+        <link rel="stylesheet" href="./styles.css">
+    </head>
+    <body>
+        <div class="navBar" id="navBar">
```

Comment:
You really shouldn't need the id "navBar" since you already have the class "navBar".